go/src/koding/kites/kloud/provider/aws/aws.go

```diff
@@ -202,10 +202,10 @@ func (meta *Cred) Valid() error {
 		return errors.New("aws: secret key is empty")
 	}
 	if !reAccessKey.MatchString(meta.AccessKey) {
-		return errors.New("aws: access key is ill-formed")
+		return errors.New("aws: access key seems wrong")
```

Comment:
> this issue is about using simpler language not about changing the meaning of it.

I know and that's why I left my comment: `ill-formed` suggests that the format of provided credential is invalid, however `invalid` may suggest that we actually checked the credentials (by calling AWS API) and we were denied.

I'm fine with `s\ill-formed\invalid`, just think that the word `format` was important here.